src/main/ngapp/package.json

```diff
@@ -29,6 +29,7 @@
     "eva-icons": "^1.1.2",
     "faker": "^4.1.0",
     "firebase": "^7.15.5",
+    "font-awesome": "^4.7.0",
```

Comment:
same here. (i am not sure whether we should be adding the json files)

Reply:
The package.json should be pushed because that keeps track of he dependencies we use